src/main/java/com/app/library/service/BookService.java

```diff
@@ -46,12 +50,13 @@ public Optional<BookDto> replaceBook(UUID bookId, BookDto bookDto) {
         return Optional.of(bookMapper.toDto(updatedBook));
     }
 
+    @Async
```

Comment:
Using async here can cause bugs, because in case of any error during delete, the user will not be notified :(

Reply:
Annotation Removed